File: MPI/mpicheck/tests/functions/4.0/MPI_Isendrecv_replace_c.f90

        
        program main
        use mpi
        
        TYPE(INTEGER) var_0(10)
       INTEGER(KIND=MPI_COUNT_KIND) var_1
       INTEGER var_2
       INTEGER var_3
       INTEGER var_4
       INTEGER var_5
       INTEGER var_6
       INTEGER var_7
       INTEGER var_8
       INTEGER var_9
        call mpi_isendrecv_replace_c(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, &
       var_8, var_9)
       call pmpi_isendrecv_replace_c(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, &
       var_8, var_9)
        end program main
    

File: MPI/mpicheck/tests/functions/4.0/MPI_Isendrecv_replace_c.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    void *var_0;
    MPI_Count var_1;
    MPI_Datatype var_2;
    int var_3;
    int var_4;
    int var_5;
    int var_6;
    MPI_Comm var_7;
    MPI_Request *var_8;
    int ret;
    /* calls */
    ret = MPI_Isendrecv_replace_c(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8);
    ret = PMPI_Isendrecv_replace_c(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8);
    return 0;
}
